### PR TITLE
chore: Document Google Drive template on docs

### PR DESCRIPTION
**Pull request by rguldener**
Document how to use Google Drive Picker API & set metadata to use the google drive pre-built integration

## Checklist before requesting a review (skip if just adding/editing APIs & templates)
- [ ] I added tests, otherwise the reason is: 
- [ ] I added observability, otherwise the reason is:
- [ ] I added analytics, otherwise the reason is:

### CODE REVIEW

File: docs-v2/integrations/all/google-drive.mdx [hunk 1]
```diff
@@ -24,8 +24,93 @@ API configuration: [`google-drive`](https://nango.dev/providers.yaml)
...
+        let filesIds: string[] = [];      
+        
+        const cid = generateCid(); // generate a unique id for the connection
+
+        try {
+        const nangoAuth = await nango.auth('google-drive', cid);
+        if (nangoAuth.connectionId) {
+            filesIds = await openGooglePicker();
+            if(filesIds.length === 0) {
+            return;
```
Comment (TBonnin) [inline docs-v2/integrations/all/google-drive.mdx]: neat: missing indentation

File: docs-v2/integrations/all/google-drive.mdx [hunk 2]
```diff
@@ -24,8 +24,93 @@ API configuration: [`google-drive`](https://nango.dev/providers.yaml)
...
+                resolve([]);
+                return;
+            }
+            else if (data.action == 'picked') {
+                const fileIds = data.docs.map((doc) => doc.id);
+                resolve(fileIds);
+            }
+            }
+        });
+        });
```
Comment (TBonnin) [inline docs-v2/integrations/all/google-drive.mdx:81]: there is some missing indentation in this function body

File: docs-v2/integrations/all/google-drive.mdx [hunk 3]
```diff
@@ -24,8 +24,93 @@ API configuration: [`google-drive`](https://nango.dev/providers.yaml)
...
+    Pass the user selected file & folder IDs to your backend.
+
+    From there, call the [Nango SDK](https://docs.nango.dev/reference/sdks/node#edit-connection-metadata) or [REST API](https://docs.nango.dev/reference/api/connection/update-metadata) to store the id's on the user's connection (SDK example):
+    ```ts
+    await nango.updateMetadata(
+        'google-drive',
+        'CONNECTION-ID',
+        {
+            'folders': ['abc', 'def', '<folder-ids>'],
+            'files': ['<file>', '<ids>', '<here>']
```
Comment (TBonnin) [inline docs-v2/integrations/all/google-drive.mdx]:
```suggestion
             'folders': ['<folder-id>, ...],
            'files': ['<file-id>', ...] 
```

**Review by TBonnin — APPROVED** [review]
Some indentations issue but lgtm otherwise

File: docs-v2/integrations/all/google-drive.mdx (hunk 2 above)
Comment (rguldener) [reply docs-v2/integrations/all/google-drive.mdx:81]: Thanks, fixed

File: docs-v2/integrations/all/google-drive.mdx (hunk 1 above)
Comment (rguldener) [reply docs-v2/integrations/all/google-drive.mdx]: fixed in the new version

**Pull request merged**